lib/postgrex.ex

```diff
@@ -79,7 +79,8 @@ defmodule Postgrex do
     * `:handshake_timeout` - Connection handshake timeout in milliseconds
       (defaults to `:timeout` value);
     * `:ssl` - Set to `true` if ssl should be used (default: `false`);
-    * `:ssl_opts` - A list of ssl options, see ssl docs;
+    * `:ssl_opts` - A list of ssl options, see the
+    [ssl docs](http://erlang.org/doc/man/ssl.html);
```

Comment:
good call. How about linking to particular function:

```markdown
* `:ssl_opts` - A list of ssl options, see `:ssl.connect/2`;
```

?

Reply:
I agree that more precise documentation would be more helpful, it's just that as this is my first PR into this project I was unsure if my suggestions was too specific.

From my understanding, the connect options the `ssl_opts` expects is actually [`tls_client_option`](http://erlang.org/doc/man/ssl.html#type-tls_client_option). 

Would this be accurate?

Reply:
Yup, let's go with that :+1:.